cfgov/regulations3k/copyable_modeladmin.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import unicode_literals
+
+from django.conf.urls import url
+from django.contrib.admin.utils import quote
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import redirect
+from django.utils.decorators import method_decorator
+
+from wagtail.contrib.modeladmin.views import InstanceSpecificView
+
+from treemodeladmin.helpers import TreeButtonHelper
+from treemodeladmin.options import TreeModelAdmin
+from treemodeladmin.views import TreeViewParentMixin
+
+
+class CopyButtonHelper(TreeButtonHelper):
+
+    def copy_button(self, pk):
+        cn = self.edit_button_classnames
+        return {
+            'url': self.url_helper.get_action_url('copy', quote(pk)),
+            'label': 'Copy',
+            'classname': cn,
```

Comment:
cn is an empty list. If you set 'classname' to the following, the button will match its siblings:

```
'classname': "button button-small button-secondary",
```

<img width="228" alt="sibling_buttons" src="https://user-images.githubusercontent.com/515885/43344520-0a6aea62-91b8-11e8-9d2b-03faa1b44071.png">

Reply:
I seem to have broken that getting the superfluous modeladmin heavy-customization code out of the way.